Exploration Queries/IoT/Process_byIoTDevice.yaml

```diff
@@ -0,0 +1,24 @@
+﻿Id: 1f3ecde7-5c69-4d44-ac93-5feac6d1cd2f
+DisplayName: Most frequent command executions on the device
+Description: Most frequent command executions on the device
+InputEntityType: IoTDevice
+InputFields:
+  - DeviceId
+  - IoTHub
+OutputEntityTypes:
+  - Process
+QueryPeriodBefore: 30m
+QueryPeriodAfter: 30m
+DataSources: SecurityIoTRawEvent
+query: |
+
+  let Process_byIoTDevice = (v_IotDevice_DeviceId:string, v_IoTDevice_IoTHub:string){
+  SecurityIoTRawEvent 
+  | where RawEventName == 'ProcessCreate'
+  | where AssociatedResourceId == parse_json(v_IoTDevice_IoTHub)['ResourceId'] and DeviceId == v_IotDevice_DeviceId
```

Comment:
I don't think this parse will work.  the AssociatedResourceId value in the data is a simply string.  for example - /subscriptions/<guid>/resourcegroups/test-rg/providers/microsoft.devices/iothubs/testhub-iothub.  A simple endswith should match the iothub name of "testhub-iothub".  | where AssociatedResourceId endswith v_IoTDevice_IoTHub or something similar

Reply:
As discussed by email, you are correct, will fix

Reply:
Thanks much!